frontend/packages/dev-console/src/components/import/advanced/AdvancedSection.tsx

```diff
@@ -45,13 +46,17 @@ const AdvancedSection: React.FC<AdvancedSectionProps> = ({ values }) => {
           </ProgressiveListItem>
         )}
         {/* Hide Deployment for Serverless */}
-        {values.serverless.enabled ? null : (
+        {values.resources !== Resources.Serverless && (
           <ProgressiveListItem name="Deployment Configuration">
```

Comment:
As updated on the UX Doc... Please do a check against the resources and change the name of this section:

```suggestion
           <ProgressiveListItem name={Resources.Kubernetes ? 'Deployment' : 'Deployment Config'}>
```

Reply:
done